docs/governance.rst

```diff
@@ -0,0 +1,247 @@
+
+  +--------------------------------------+
+  | `Logan Dowdle`_ (@dowdlelt)              | 
+  +--------------------------------------+
+  | `Elizabeth DuPre`_ (@emdupre)           |
+  +--------------------------------------+
+  | `Dan Handwerker`_ (@handwerkerd)   |
+  +--------------------------------------+
+  | `Taylor Salo`_ (@tsalo)                           |
+  +--------------------------------------+
+  | `Joshua Teves`_ (@jbteves)                  |
+  +--------------------------------------+
+  | `Eneko Uruñuela`_ (@eurunuela)          |
+  +--------------------------------------+
+
+Focused Leadership Roles
+  We have identified key responsibilities or skills that help advance tedana
+  development and created roles for each of these responsibilities. One
+  person can fill more than one role and more than one person can decide to
+  share or split the responsibilities of a role. Any contributor can propose
+  the creation of new focused leadership roles.  A person can take on a
+  leadership role without being a Maintainer or Steering Committee member
+
+
+  - | Task manager & record keeper: `Dan Handwerker`_
+    |   Helps write & keep track of notes from meetings
+    |   Keeps track of issues or items that should be addressed
+    |   Follows up with people who volunteered to address an item or alerts the broader community of known tasks that could use a volunteer
+  - | MR physics leader `César Caballero-Gaudes`_
+    |   Someone who can make sure calculations fit within our understanding of MR physics
+    |   Someone who can either answer MRI physics questions related to multi-echo or direct people to where they can find answers
+  - | Processing algorithms leader `Eneko Uruñuela`_ (Decomposition) & `Dan Handwerker`_ (Metrics & Decision Tree)
+    |   Someone who can make sure algorithms are appropriately implemented (or knows enough to delegate to someone who can make sure implementation is good)
+    |   Someone who can either answer processing algorithm questions or direct people to where they can find answers
+  - | Collaborative programming leader `Elizabeth DuPre`_ & `Joshua Teves`_
+    |   Helps make sure tedana is following best practices for Python code design, testing, and communication for issues/pull requests etc.
+  - | Communications leader `Joshua Teves`_
+    |   Mailing list manager & other outward-facing communication about the project
+  - | New contributors leader `Taylor Salo`_
+    |   Leads efforts to make contributor documentation more welcoming
+    |   Is a point of contact for potential contributors to make them feel welcome and direct them to relevant resources or issues
+  - | Multi-echo fMRI support leader `Logan Dowdle`_
+    |   Monitors places where people may ask questions about tedana or multi-echo fMRI and tries to find someone to answer those questions
+  - | Enforcer(s) of the `code of conduct`_ `Elizabeth DuPre`_ & `Dan Handwerker`_ & another volunteer
```

Comment:
Are we adding @smoia here?

Reply:
@smoia if you're willing to that would be great, could you confirm that you'd be interested?